httpserver/service_node.cpp

```diff
@@ -26,6 +26,50 @@ using service_node::storage::Item;
 namespace loki {
 using http_server::connection_t;
 
+constexpr std::array<std::chrono::seconds, 5> RETRY_INTERVALS = {
+    std::chrono::seconds(5), std::chrono::seconds(10), std::chrono::seconds(20),
+    std::chrono::seconds(40), std::chrono::seconds(80)};
+
+FailedWork::FailedWork(boost::asio::io_context& ioc, const sn_record_t& sn,
+                       const std::shared_ptr<request_t> req)
```

Comment:
missing reference operator `&` ?

Reply:
The intention here is to copy into `request_`. One way to do it is to pass by value and then move (I might need to remove `const` though). I don't want to move out of the original value.

Reply:
This seems relevant: https://herbsutter.com/2013/06/05/gotw-91-solution-smart-pointer-parameters/ (part e)

Reply:
Yeah removing the const makes it more clear, thanks!